src/lottie/lottieitem.cpp

```diff
@@ -830,16 +830,18 @@ DrawableList LOTShapeLayerItem::renderList()
 bool LOTContentGroupItem::resolveKeyPath(LOTKeyPath &keyPath, uint depth,
                                          LOTVariant &value)
 {
-    if (!keyPath.matches(name(), depth)) {
```

Comment:
I think you should make a separate patch for this change.

Reply:
I dont understand. 
This is change that occurred when the mModel was added.

Reply:
ok, got it.